tests/integration/testsuites/gateway/features/gateway.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Checking default kyma gateway
+  Scenario: Kyma gateway is deployed
+    Given there is an APIGateway operator in "Ready" state
```

Comment:
We don't apply the APIGateway, nor do we remove it after a test. This means we don't know how our APIGateway is configured. The precondition is not part of the Setup of the scenario. I'd not have a central setup APIGateway for all tests. That can easily cause tests being dependant on some order or being flaky. We can also not be sure that the test runs correctly, since we don't start from a "fresh" APIGateway for this scenario.